contracts/Proxy.sol

```diff
@@ -17,19 +17,20 @@ contract Proxy is Proxied {
     }
 
     /// @dev Fallback function forwards all transactions and returns all received return data.
-    function() external payable {
+    fallback() external payable {
         address _masterCopy = masterCopy;
         assembly {
-            calldatacopy(0, 0, calldatasize)
-            let success := delegatecall(not(0), _masterCopy, 0, calldatasize, 0, 0)
-            returndatacopy(0, 0, returndatasize)
+            calldatacopy(0, 0, calldatasize())
+            let success := delegatecall(not(0), _masterCopy, 0, calldatasize(), 0, 0)
+            returndatacopy(0, 0, returndatasize())
             switch success
                 case 0 {
-                    revert(0, returndatasize)
+                    revert(0, returndatasize())
                 }
                 default {
-                    return(0, returndatasize)
+                    return(0, returndatasize())
                 }
         }
     }
+
```

Comment:
Extra newline added by accident?

Does not having the `receive` function produce a warning?

Reply:
Yes it produces the warning, I have pasted already in the above discussion.
```suggestion
```

Reply:
https://github.com/gnosis/util-contracts/pull/40#discussion_r533228195